cypress.config.js

```diff
@@ -14,9 +14,19 @@ module.exports = defineConfig({
         generateUser() {
           const randomNumber = Math.ceil(Math.random(1000) * 1000);
```

Comment:
The use of `Math.random(1000)` is incorrect. `Math.random()` generates a random number between 0 and 1, so multiplying it by 1000 and then using `Math.ceil()` will not produce a number between 1 and 1000 as intended. You should use `Math.floor(Math.random() * 1000) + 1` to achieve this.